Documentation/gettingstarted/istio.rst

```diff
@@ -96,7 +96,9 @@ To deploy Cilium, run:
 
     .. parsed-literal::
 
-      $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes/1.7/cilium-sidecar.yaml
+      $ curl -s \ |SCM_WEB|\/examples/kubernetes/1.7/cilium.yaml | \\
+        sed -e 's/sidecar-http-proxy: "false"/sidecar-http-proxy: "true"/' | \\
+        kubectl create -f
```

Comment:
please add a new line here otherwise the users might get confused

Reply:
Done.

Reply:
does the `-f` need a `-`? I'm assuming it works but the help doesn't mention the behaviour without the `-`